HHS/Head-Start-TTADP
# [TTAHUB-2318] goal & objective collaborators

## Description of change

- Record collaborators for goal & objectives
- Populate all existing goals & objectives
- Collaborator types:
  - Creator - The user who creates a goal or objective
  - Editor - The user(s) who edits the text of a goal or objective
  - Linker - The user who links the goal or objective to a report
  - Utilizer - The user who created report and users listed as collaborators on report.

- added db error log monitoring to circleci builds

## How to test
Create or edit a goal, check the db looking at the GoalCollaborators and ObjectiveCollaborators.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2318


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
